assets/stylesheets/layout/_main.scss

```diff
@@ -275,54 +280,58 @@ html.iframed {
 $sidebar-width-max: 272px;
 $sidebar-width-min: 228px;
 
+.wp {
+	display: flex;
+	flex-direction: column;
+	position: fixed;
```

Comment:
Do we need this? Would it cause performance issues to have this element fixed positioned?